inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_reshape.cpp

```diff
@@ -50,9 +78,19 @@ std::vector<ReshapeTestParams> reshapeTestParams = {
         std::make_tuple(DataShapeWithUpperBound{{750, 1}, {750, 1}}, true, ShapeDescriptor{-1, 1, 1, 1}),
         std::make_tuple(DataShapeWithUpperBound{{750, 4}, {1000, 4}}, true, ShapeDescriptor{1, -1, 4}),
         std::make_tuple(DataShapeWithUpperBound{{750}, {1000}}, true, ShapeDescriptor{1, 1, -1}),
+        std::make_tuple(DataShapeWithUpperBound{{800, 81, 4}, {1000, 81, 4}}, true, ShapeDescriptor{0, -1}),
+        std::make_tuple(DataShapeWithUpperBound{{800, 256, 7, 7}, {1000, 256, 7, 7}}, true, ShapeDescriptor{0, -1}),
 };
 
-INSTANTIATE_TEST_CASE_P(smoke_DynamicReshape, DSR_Reshape,
+INSTANTIATE_TEST_CASE_P(smoke_DynamicReshape, DSR_ReshapeWithStaticDescriptor,
+    ::testing::Combine(
+        ::testing::Values(ngraph::element::f16,
```

Comment:
Nitpick: precisions may be extracted there and reused in the next instantiation.

Reply:
Done